FIRO_TSEnsembles/src/main/java/hec/stats/InlineStats.java

```diff
@@ -19,12 +20,16 @@ public int getSampleSize(){
         return _sampleSize;
     }
     public float getSampleVariance(){
-        return _mean;
+        return _sampleVariance;
     }
+    public float getStandardDeviation() {return _standardDeviation; }
```

Comment:
call sqrt on sample variance here.

technically i think you have a degrees of freedom issue here, you may have to multiply by n/n-1  on sample variance first.

Reply:
updated based on recommendation